ansible/roles/update_machines/tasks/reboot.yaml

```
@@ -0,0 +1,18 @@
+
+- name: Reboot the machine
+- name: Reboot the machine
+  ansible.builtin.reboot:
+    msg: "System reboot due to package updates"
+    pre_reboot_delay: 30
+    post_reboot_delay: 60
+    reboot_timeout: 600
+    test_command: uptime
+  when: >
+    (reboot_check is defined and reboot_check.changed) or
+    ansible_facts['os_family'] == 'Alpine'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)